backend/test/private-api/private-api.api-tokens.e2e-spec.ts

```diff
@@ -0,0 +1,172 @@
+        const response = await agentUser1
+          .post(`${PRIVATE_API_PREFIX}/tokens`)
+          .send({
+            label: tokenName,
+          })
+          .expect('Content-Type', /json/)
+          .expect(201);
+        apiTokenWithSecret = response.body;
+      });
+    });
+    it("guest users can't create api tokens", async () => {
+      await agentGuestUser
+        .post(`${PRIVATE_API_PREFIX}/tokens`)
+        .send({
+          label: tokenName,
+          validUntil: 0,
+        })
+        .expect('Content-Type', /json/)
+        .expect(401);
+    });
+    it("non logged-in users can't create api tokens", async () => {
+      await agentNotLoggedIn
+        .post(`${PRIVATE_API_PREFIX}/tokens`)
+        .send({
+          label: tokenName,
+          validUntil: 0,
+        })
+        .expect('Content-Type', /json/)
+        .expect(401);
+    });
+  });
+
+  describe(`GET ${PRIVATE_API_PREFIX}/tokens`, () => {
+    const tokenName = 'test';
+    beforeEach(async () => {
+      await testSetup.apiTokenService.createToken(
+        testSetup.userIds[0],
+        tokenName,
+      );
+    });
+    it('owner can see api tokens', async () => {
+      const response = await agentUser1
+        .get(`${PRIVATE_API_PREFIX}/tokens/`)
+        .expect('Content-Type', /json/)
+        .expect(200);
+      expect(response.body[0].label).toBe(tokenName);
+      expect(new Date(response.body[0].validUntil).getTime()).toBeGreaterThan(
+        Date.now(),
+      );
+      expect(response.body[0].lastUsedAt).toBe(null);
+      expect(response.body[0].secret).not.toBeDefined();
+    });
+    it("other user can't see api tokens", async () => {
+      const response = await agentUser2
+        .get(`${PRIVATE_API_PREFIX}/tokens/`)
+        .expect('Content-Type', /json/)
+        .expect(200);
+      expect(response.body).toHaveLength(1);
+    });
+    it("guest user can't see api tokens", async () => {
+      await agentGuestUser
+        .get(`${PRIVATE_API_PREFIX}/tokens/`)
+        .expect('Content-Type', /json/)
+        .expect(401);
+    });
+    it("non logged-in user can't see api tokens", async () => {
+      await agentNotLoggedIn
+        .get(`${PRIVATE_API_PREFIX}/tokens/`)
+        .expect('Content-Type', /json/)
+        .expect(401);
+    });
+  });
+  describe(`DELETE ${PRIVATE_API_PREFIX}/tokens/:keyid`, () => {
+    const tokenName = 'private-api-test-token';
+    let keyId: string;
+    beforeEach(async () => {
+      const token = await testSetup.apiTokenService.createToken(
+        testSetup.userIds[0],
+        tokenName,
+      );
+      keyId = token.keyId;
+    });
+    it('owner can delete api tokens', async () => {
+      await agentUser1
+        .delete(`${PRIVATE_API_PREFIX}/tokens/${keyId}`)
+        .expect(204);
+      const tokens = await testSetup.apiTokenService.getTokensOfUserById(
+        testSetup.userIds[0],
+      );
+      expect(tokens).toHaveLength(1);
```

Comment:
Why is it 1 token after deletion of the previously added token?
Is there one token already pre-existing?